webapp/graphite/readers.py

```diff
@@ -188,6 +188,86 @@ def fetch(self, startTime, endTime):
     return time_info, values
 
 
+class CarbonCacheReader(object):
+  __slots__ = ('metric')
+  supported = True
+
+  def __init__(self, metric):
+    self.metric = metric
+
+  def get_intervals(self):
+    # intervals doesn't matter in such type of reader
+    # Let's return time.time()
+    start = time.time()
+    end = start
+    return IntervalSet( [Interval(start, end)] )
+
+  def fetch(self, startTime, endTime):
```

Comment:
lots of camelCase variables here, which isn't standard in Python.

underscore_variable_names would be great.